.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -31,4 +31,4 @@ Which OS you used to reveal the bug.
 **Additional context**
 Add any other context about the problem here.
 
-Avalanche Bug Bounty program can be found [here](https://immunefi.com/bug-bounty/avalanche/information/).
+You can submit a bug on the [Avalanche Bug Bounty program page](https://hackenproof.com/avalanche/avalanche-protocol).
```

Comment:
I think this link was correct.

https://hackenproof.com/avalanche/avalanche-protocol is a valid link, but goes to a 404 page. 

https://immunefi.com/bug-bounty/avalanche/information/ goes to the correct bug bounty program, AFAICT

Reply:
Ah that's weird -- it somehow passes CI, and this is what the link was changed to in coreth. I will fix it here, and it gets fixed through deletion in coreth here: https://github.com/ava-labs/avalanchego/pull/4578, so think it can remain there for now.